src/measure/Measure.i

```diff
@@ -96,6 +96,55 @@
 };
 
 #if defined SWIGRUBY
+  %feature("director:except") {
+    // Mimic openstudio::evalString for a possible approach to reporting more info or a callstack.
```

Comment:
For ruby, mimic the https://github.com/NREL/OpenStudio/blob/c0cbe73b51f34c459d67a17da36739795bbf25fc2/ruby/interpreter/RubyEval.hpp#L18